README.md

```diff
@@ -54,10 +54,10 @@ Other parameters can also be specified as below:
 Donut help
 Usage: Donut reports [options]
 
-  -s <value> | --sourcedirs <value>
-        Use --sourcedirs cucumber:/my/path/cucumber-reports -> Required, or Use --sourcedirs specflow:/my/path/specflow-reports,/my/nunit-reports
-  -o <value> | --outputdir <value>
-        Use --outputdir /my/path/output/donut
+  -s <value> | --sourcePaths <value>

```

Comment:
Just raising some questions, let me know your thoughts.

Reply:
This is a first step to supporting different formats. The idea is that you wouldn't need to convert to a gherkin JSON format using the adaptors. You would just need to specify junit:/path/to/junit etc.

Reply:
Any number of paths to gherkin results json files are supported rather than 1 cuke and non-cuke (adapted to gherkin json from other format). However, the only supported result format is currently gherkin JSON. I'm planning on building in support for other formats directly into donut in a subsequent release.

Reply:
👍